core/lexicon/en/setting.inc.php

```diff
@@ -266,6 +266,12 @@
 $_lang['setting_failed_login_attempts'] = 'Failed Login Attempts';
 $_lang['setting_failed_login_attempts_desc'] = 'The number of failed login attempts a User is allowed before becoming \'blocked\'.';
 
+$_lang['setting_fast_search_in_content'] = 'Allow search in content';
+$_lang['setting_fast_search_in_content_desc'] = 'If \'Yes\', then the content of the element (resource, template, chunk, etc.) will also be available for fast search.';
```

Comment:
Could we use emphasizing via bolding the text instead of quotes? To get "If **Yes**, then ..."

Reply:
I think this is for another PR, because in lexicons, `Yes / No` is indicated everywhere through quotation marks.

Reply:
Yes, fixing all such texts is the other task, but if we can do it easier now, we shouldn't keep the technical debt in any case.

Reply:
I do not agree here. Because if the task (changing the quotes to bold) takes months, or will be forgotten, then we will get an inconsistency in the interface. Which is very likely...

Reply:
In this case, I agree with creating issue about it, just to not forget about it.
I don't think that changing the quotes can take months, it is mostly about the agreement, but technically it is easy.